.env.example

```diff
@@ -37,6 +37,7 @@ EXPIRE_REGISTRATION_EXEMPTION_RUN_TIME="00:05"
 # Exporter config
 EXPORT_SERVICE_BATCH_SIZE=10
 EXPORT_SERVICE_EPR_EXPORT_TIME="1:05"
+EXPORT_SERVICE_AD_RENEWAL_LETTERS_TIME="1:05"
```

Comment:
Is it OK to have this scheduled at the same time as the EPR export? Feels like they shouldn't be clashing.

Reply:
The EPR exports in prod actually run at 22.05 :) We should fix these values to reflect the production once. For another PR :)